icu4c/source/test/testdata/message2/README.txt

```diff
@@ -0,0 +1,18 @@
+The format of the JSON files in this directory follows the same format as `test-core.json`
+in the spec, described in:
+
+https://github.com/unicode-org/message-format-wg/blob/main/test/README.md
+
+The `parts` and `cleanSrc` fields are not used.
+
+Additional "char" and "line" fields may be present with integer values,
+used for tests expected to trigger a syntax error.
+If present, "char" reflects the expected character offset and "line"
+reflects the expected line number in the parse error.
+The files with "diagnostics" in the name have these fields filled in.
+
+An additional `comment` field may be present, which is only for human readers.
+
+Tests in the `spec/` subdirectory are taken from https://github.com/unicode-org/message-format-wg/blob/main/test
+and need to be manually updated if the contents change upstream.
+
```

Comment:
```suggestion

Copyright © 2024 Unicode, Inc. Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the United States and other countries.
```

(remember to accept the comment, if you want to accept it this way, with `ICU-22746` as the beginning of the commit messag)